### PR TITLE
Build a single font-face family

### PR DESCRIPTION
**Pull request by GregoireCiles**
## Proposed changes

To be able to build a single family of font-faces, 
It's must to check whether the `$font-faces` list is separated by `spaces` or by `commas`.

---

For example:
```scss
$font-faces: (
    "Webfont Sans" "webfont-sans_regular" 400 normal
)
```

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)

---

## Editorial note

1. Fixes support for building a single font family using `$font-faces` by fixing iteration of that global variable in the `font-faces` mixin.

   Because Sass accepts commas, spaces, and slashes as list separators, when `$font-faces` contained only one family (`<font-name> <font-file-basename> <font-weight> <font-style>`), it would iterate each segment of the family instead of the tuple as a whole.

### CODE REVIEW

**Review by mcaskill — CHANGES_REQUESTED** [review]
Thank you for catching this quirk of Sass.
We had not stumbled upon yet 😅

File: assets/styles/tools/_fonts.scss [hunk 1]
```diff
@@ -13,7 +13,7 @@
         font-display: swap;
         font-family: nth($webfont, 1);
         src: url("#{$dir}#{nth($webfont, 2)}.woff2") format("woff2"),
-             url("#{$dir}#{nth($webfont, 2)}.woff") format("woff");
+            url("#{$dir}#{nth($webfont, 2)}.woff") format("woff");
```
[the remark this answers is not in the document]
Comment (GregoireCiles) [reply assets/styles/tools/_fonts.scss]: Sorry it's prettier 😅

**Review by mcaskill — CHANGES_REQUESTED** [review]

File: assets/styles/tools/_fonts.scss [hunk 2]
```diff
@@ -26,8 +26,14 @@
...
-    @each $webfont in $webfonts {
-        @include font-face($webfont, $dir);
+    @if (list-separator($webfonts) == "comma") {
+        @each $webfont in $webfonts {
+            @include font-face($webfont, $dir);
+        }
+    } @else if (list-separator($webfonts) == "space") {
+        @include font-face($webfonts, $dir);
+    } @else {
+        @error 'Invalid list separator for $webfonts';
```
[the remark this answers is not in the document]
Comment (GregoireCiles) [reply assets/styles/tools/_fonts.scss:36]: Allowing everyone to choose their own delimiter is much better, I hadn't thought of that directly!

[the remark this answers is not in the document]
Comment (mcaskill) [reply assets/styles/tools/_fonts.scss:36]: Neither did I at first. Given Sass' many accepted delimiters, this should be a safer option.

[the remark this answers is not in the document]
Comment (GregoireCiles) [reply assets/styles/tools/_fonts.scss]: Yeah for sure !

**Review by mcaskill — APPROVED** [review]

**Comment by mcaskill** [comment]
@GregoireCiles Thanks! I've [merged this pull request](https://github.com/locomotivemtl/locomotive-boilerplate/commit/de6b3d73a14699bcf7dfddee59923a4aed9888ba) but while fixing a rebase conflict, it appears to have lost reference to this pull request.

Maybe if you rebase your `fix/scss-build-font-faces` branch to e8af2200, it will mark this branch as merged.

**Pull request closed without merging**